### DISCUSSION TITLE
next/image width 100% with layout "fixed"

### DISCUSSION THREAD

**Discussion by Thunderfu** [vercel/next.js#19790] [Help]
I'm not able to set 100% width in the next/image, when set, the width is 100px.

```
<Image
    width="100%"
    height={160}
    objectFit="contain"
    layout="fixed"
/>
```

I need the width 100% because the parent is responsive but I don't know what the width is.

**Comment by deadcoder0904** [comment]
Hey @Thunderfu I don't think that's possible. See this comment → https://github.com/vercel/next.js/discussions/18947#discussioncomment-133008
